pkg/service/query_test.go

```diff
@@ -65,6 +65,7 @@ func TestQueryServiceNodePortWithIngress(t *testing.T) {
 	assert.Len(t, svc.Spec.Ports, 1)
 	assert.Equal(t, int32(16686), svc.Spec.Ports[0].Port)
 	assert.Equal(t, "http-query", svc.Spec.Ports[0].Name)
+	assert.Equal(t, int32(0), svc.Spec.Ports[0].NodePort)
```

Comment:
```suggestion
	assert.Equal(t, 0, svc.Spec.Ports[0].NodePort)
```

Reply:
This is will cause the test to fail, since the value of `t` is expected to be an `int32` value. While the values themselves will be equal, the `assert.Equal` statement will fail due to a type mismatch.